docs/source/extension/extension_migration.rst

```diff
@@ -60,6 +60,11 @@ bumped their major version (following semver convention). We want to point out p
    The search provider API has been fully reworked. But the logic is similar, for new type of documents
    you will need to register a ``ISearchProviderFactory`` to the ``ISearchProviderRegistry``. The
    factory will build a ``ISearchProvider`` for the document widget.
+- ``@jupyterlab/extensionmanager`` from 3.x to 4.x
+   The frontend API has been drastically reduce to fetch all information from the backend. And it is now advised
+   to provide a custom ``ExtensionsManager`` class for your need rather than overriding the frontend plugins.
+   See ``jupyterlab/extensions/pypi.py`` for an example using PyPI.org and pip. You can then register your manager
+   defining an entry point in the Python package defining it; see ``pyproject.toml::project.entry-points."jupyterlab.extension_manager_v1"``.
```

Comment:
Maybe something like the following to avoid repeating "defining"?

```suggestion
   by defining an entry point in the Python package; see ``pyproject.toml::project.entry-points."jupyterlab.extension_manager_v1"``.
```